src/System.Runtime/tests/TimeZoneInfoTests.netstandard1.7.cs

```diff
@@ -0,0 +1,274 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class TimeZoneInfoTests
+    {
+        [Fact]
+        public static void ClearCachedData()
+        {
+            TimeZoneInfo cst = TimeZoneInfo.FindSystemTimeZoneById("Central Standard Time");
+            TimeZoneInfo local = TimeZoneInfo.Local;
+
+            TimeZoneInfo.ClearCachedData();
+            Assert.ThrowsAny<ArgumentException>(() =>
+            {
+                TimeZoneInfo.ConvertTime(DateTime.Now, local, cst);
+            });
+        }
+
+        [Fact]
+        public static void LibyaTimeZone_TimeZoneNotFoundException()
```

Comment:
I would remove this test.  Here's my thinking.  If `TimeZoneInfo.FindSystemTimeZoneById(s_strLibya)` throws, this test isn't going to execute.  The value this test provides is in ensuring the LibyaTimeZone test works as expected and in general we don't expect the exception to be thrown.  We already have functional coverage from the test which catches Exception.